rct229/rules/section6/section6rule4.py

```diff
@@ -0,0 +1,134 @@
+from rct229.data.schema_enums import schema_enums
+from rct229.data_fns.table_G3_7_fns import table_G3_7_lookup
+from rct229.rule_engine.rule_base import (
+    RuleDefinitionBase,
+    RuleDefinitionListIndexedBase,
+)
+from rct229.rule_engine.user_baseline_proposed_vals import UserBaselineProposedVals
+from rct229.ruleset_functions.get_avg_zone_ht import get_avg_zone_ht
+from rct229.ruleset_functions.get_building_segment_lighting_status_type_dict import (
+    LightingStatusType,
+    get_building_segment_lighting_status_type_dict,
+)
+from rct229.utils.jsonpath_utils import find_all
+from rct229.utils.std_comparisons import std_equal
+
+OFFICE_OPEN_PLAN = schema_enums[
+    "LightingSpaceType2019ASHRAE901TG37"
+].OFFICE_OPEN_PLAN.name
+
+
+class Section6Rule4(RuleDefinitionListIndexedBase):
+    """Rule 4 of ASHRAE 90.1-2019 Appendix G Section 6 (Lighting)"""
+
+    def __init__(self):
+        super(Section6Rule7, self).__init__(
```

Comment:
I believe ```super(Section6Rule7, self).__init__(``` should be ```super(Section6Rule4, self).__init__(``` or class name should be ```Section6Rule7``` to be consistent. Would you be able to verify this?